### ISSUE TITLE
Car Bug

### ISSUE THREAD

**Issue by blendt12** [Everfall-Development/ef-issues#4]
Viktor Petrov

After upgrading my vehicle at Hayes, I have no headlights. Shop unable to fix. Not tint or cosmetics.

**Comment by jellyton255** [comment]
Do you continue to not have headlights over restart?

**Comment by blendt12** [comment]
Yes. They are completely missing. They appear to show inside of my car near
my rear tires

**Comment by jellyton255** [comment]
I'm aware of the issue and how to fix it, but I'm not certain what is causing it.
